ozaria/site/components/common/LayoutChrome.vue

```diff
@@ -11,66 +10,71 @@
       <img id="btn-top" src="/images/ozaria/layout/chrome/Chrome_Button.png" />
       <img id="btn-bottom" src="/images/ozaria/layout/chrome/Chrome_Button.png" />
     </div>
+
+    <div class="chrome-container">
```

Comment:
Added this so we can maintain proper padding for all components wrapped inside of chrome container.  The padding does not yet work due to some of the positioning types / their behavior in CSS.

Without proper padding the chrome UI will overlap whatever it is surrounding, which could possibly result in parts of the UX being invisible / unusable.  Is this a known issue or is this intentional?  If it's a known issue I can resolve here as it needs to be resolved for interactives.

Reply:
It is currently intentional that the chrome overlaps the contents.
There have been different instructions for how to handle the chrome regarding Cinematics and Cutscenes.
 - Cinematics intentionally get the edges covered up.
 - Cutscenes will have padding to ensure the video controls aren't obscured.

How should interactives interact with the chrome?

Reply:
Going to revert my changes and use master version for now.  Will follow up in Slack convo started earlier today to determine how to handle this / follow up with PR.